### DISCUSSION TITLE
How could I have all calendar groups selected by default?

### DISCUSSION THREAD

**Discussion by OscarGS86** [YetiForceCompany/YetiForceCRM#15387] [How to]
Hi guys,

I have different calendar groups and I would like to know how I can have them all selected by default?

Now, I need to select one by one and It is a bit painful.

Could you help me to get all group's calendars selected by default when I access the calendar section?

![Screenshot 2021-10-29 at 11 46 33](https://user-images.githubusercontent.com/92228499/139414336-521699fd-58b6-4e9a-9379-30bb9e9b623d.png)

Cheers,

**Comment by OscarGS86** [comment]
Hi guys,

Does anyone know how to select all calendars by default? 

Your help is much appreciated. 

Thank you

**Comment by OscarGS86** [comment]
Hey community,

Any help on this one?

Thanks

**Comment by Cattrah** [comment]
Have you found any solutions to this? I'd like to have all the user calendars selected by default in the calendar filter, similar situation. Closest I got was find this file /yeti/layouts/basic/modules/Calendar/Standard/RightPanel.tpl seems to create all the Users and Groups in the filters and select the logged in user, but when editing this file, it doesn't seem to reflect any changes. Not sure where else to look.

**Comment by rskrzypczak** [comment]
![image](https://user-images.githubusercontent.com/10221999/165673359-e919f90c-5876-4cb1-b655-f570563a3604.png)
This has been added to the development version and will be available in version 6.4.
However, you won't be able to check it right now as fullcalendar is being updated and the calendar view may just not work.

**Reply by Cattrah** [reply]
Fantastic! Thank you! Looking forward to this update.
